assets/src/blocks/Covers/CoversImagePlaceholder.js

```diff
@@ -1,7 +1,7 @@
 import { ImagePlaceholderIcon } from '../../components/ImagePlaceholderIcon';
 
-export const CoversImagePlaceholder = ({ height }) => (
-  <div className='covers-image-placeholder' style={{ height }}>
-    <ImagePlaceholderIcon width={20} height={20} fill={'#ffffff'}/>
+export const CoversImagePlaceholder = ( { height } ) => (
+  <div className="covers-image-placeholder" style={ { height } }>
+    <ImagePlaceholderIcon width={ 20 } height={ 20 } fill={ '#ffffff' } />
```

Comment:
Not a big deal, but here I think that we could directly write `fill="#ffffff"`

Reply:
Done.